eth/v1alpha1/validator.proto

```diff
@@ -15,13 +15,119 @@ syntax = "proto3";
+    // Validator duties should be polled every epoch, but due to chain reorg of >MIN_SEED_LOOKAHEAD could occur,
+    // the validator duties could chain. For complete safety, it is recommended to poll at every slot to ensure
+    // validator is fully aware of any sudden chain reorg.
+    rpc Duties(DutiesRequest) returns (DutiesResponse) {
+        option (google.api.http) = {
+            get: "/eth/v1alpha1/validator/duties"
+        };
+    }
+
+    // Retrieves the latest valid beacon block to be proposed on the beacon chain.
+    //
+    // The server returns a new beacon block, without proposer signature, that can be
+    // proposed on the beacon chain. The block should be filled with all the necessary
+    // data for proposer to sign.
+    rpc Block(BlockRequest) returns (Block) {
+        option (google.api.http) = {
+            get: "/eth/v1alpha1/validator/block"
+        };
+    }
+
+    // Sends the newly signed beacon block to beacon node to be broadcasted to the rest of the network.
+    //
+    // The validator sends the newly signed beacon block to the beacon node so the beacon block can
+    // be included in the beacon chain. The beacon node is expected to validate and process the
+    // beacon block into its state, however if the beacon block fails validator, it will still
+    // be broadcasted to the rest of the network but a different status code is returned.
+    rpc ProposeBlock(Block) returns (google.protobuf.Empty) {
+        option (google.api.http) = {
+            post: "/eth/v1alpha1/validator/block"
+        };
+    }
+
+    // Retrieves the latest valid attestation to be attested on the beacon chain.
+    //
+    // The server returns the latest valid attestation data which represents the correct vote
+    // for the head of the beacon chain,
+    rpc Attestation(AttestationRequest) returns (AttestationData) {
+        option (google.api.http) = {
+            get: "/eth/v1alpha1/validator/attestation"
+        };
+    }
+
+    // Sends the newly signed attestation to beacon node to be broadcasted to the rest of the network.
+    //
+    // The validator sends the newly signed attestation to the beacon node for the attestation to
+    // be included in the beacon chain. The beacon node is expected to validate and aggregate the
+    // attestations into the state, if the attestation fails validator, it'd still be broadcasted but a
+    // different status code is returned.
+    rpc ProposeAttestation(Attestation) returns (google.protobuf.Empty) {
+        option (google.api.http) = {
+            post: "/eth/v1alpha1/validator/attestation"
+        };
+    }
+}
+
+message DutiesRequest {
+    // Epoch at which validators should perform their duties.
+    uint64 epoch = 1;
+    // Array of byte encoded BLS public keys.
+    repeated bytes public_keys = 2;
+}
+
+message DutiesResponse {
+    repeated Duty duties = 1;
+    message Duty {
+        bytes pub_key = 1;
```

Comment:
Please use `public_key` as that's how we call this else where.
https://github.com/prysmaticlabs/ethereumapis/blob/0b6fad5ffca44741a024c71d5dc53e235c691232/eth/v1alpha1/beacon_chain.proto#L218

Reply:
Please add a field description and specify that this is a 96 byte public key for the validator who's duty this is.